tests/unit/test_parser.py

```diff
@@ -973,7 +973,10 @@ def test_python_model_parse(self):
             raw_code=python_model,
             checksum=block.file.checksum,
             unrendered_config={"materialized": "table", "packages": python_packages},
-            config_call_dict={"materialized": "table", "packages": python_packages},
+            config_call_dict={
```

Comment:
In order to accept this PR, we will need an effective test.

Reply:
Yes! The issue was that `test_python_model_custom_materialization` was testing with `materialized = view`, which is the default value, so nothing was actually tested.  I have updated the PR